src/widgets/reflow.rs

```diff
@@ -124,16 +125,19 @@ pub struct LineTruncator<'a, 'b> {
     symbols: &'b mut dyn Iterator<Item = Styled<'a>>,
     max_line_width: u16,
     current_line: Vec<Styled<'a>>,
+    cut: u16,
 }
 
 impl<'a, 'b> LineTruncator<'a, 'b> {
     pub fn new(
         symbols: &'b mut dyn Iterator<Item = Styled<'a>>,
         max_line_width: u16,
+        cut: u16,
```

Comment:
Given that it is not a required property, could we default to `0` and have a different method to set it ?

Reply:
That's a good idea.